examples/csv_mail_merge/index.js

```diff
@@ -0,0 +1,80 @@
+'use strict';
+
+var converter = require('json-2-csv');
+var fs       = require('fs');
+var moment   = require('moment');
+var parse    = require('csv-parse');
+
+var lobFactory = require('../../lib/index.js');
```

Comment:
Yeah for the examples it is best to require the relative path because then you don't need to worry about the user having a different version of the wrapper installed on their machine which may not be compatible